cityofaustin/atd-vz-data
# Use `crash_timestamp` and `prsn_death_timestamp` columns

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/17954

This ended up being a bit more of a lift than i was expecting, but the majority of this diff is just codegen and linting 😅 

## Testing
1. Start your data model stack as usual and run the CRIS import ETL.
2. Check that all of the VZE pages load and interact normally
3. Pick a few crashes at random and compare the crash date you see locally versus in prod. They should match if you're testing from US/Central. Otherwise you will notice that the crash date is always rendered in local time. (That change is not specific to this PR, but I have fixed up the formatting of the timestamp strings).
4. Query the people table and compare  `prsn_death_date` and `prsn_death_time` to  `prsn_death_timestamp`
---
#### Ship list
